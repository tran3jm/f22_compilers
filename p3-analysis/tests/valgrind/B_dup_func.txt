==1718682== Memcheck, a memory error detector
==1718682== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1718682== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==1718682== Command: ../decaf inputs/dup_func.decaf
==1718682== 
Duplicate symbols named 'main' in scope started on line 1
==1718682== 
==1718682== HEAP SUMMARY:
==1718682==     in use at exit: 0 bytes in 0 blocks
==1718682==   total heap usage: 1,817 allocs, 1,817 frees, 221,668 bytes allocated
==1718682== 
==1718682== All heap blocks were freed -- no leaks are possible
==1718682== 
==1718682== For lists of detected and suppressed errors, rerun with: -s
==1718682== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
