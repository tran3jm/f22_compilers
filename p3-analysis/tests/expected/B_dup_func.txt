Duplicate symbols named 'main' in scope started on line 1
